### PR TITLE
add direction argument to scale_*_tableau()

### LINKED ISSUE #111: Add "direction" argument to scale_*_tableau()
Is it possible to add `direction = 1` argument to `scale_fill_tableau()` and `scale_color_tableau()` and pass it to `tableau_color_pal()`, which already accepts it? It would be consistent with `ggplot2::scale_*_brewer()`.

If interested, I could give a PR a try. Seems fairly simple.

### PR DESCRIPTION
**Pull request by vadimus202**
Fixes #111. Thanks

### CODE REVIEW

File: R/tableau.R [hunk 1]
```diff
@@ -63,7 +64,6 @@ tableau_color_pal <- function(palette = "Tableau 10",
 #' and \funclink{scale_colour_gradient2_tableau} for the diverging continuous
 #' color scales from Tableu.
 #'
-#' @inheritParams ggplot2::scale_colour_hue
```
Comment (jrnold) [inline R/tableau.R]: Why did you drop this?

**Comment by jrnold** [comment]
Looks good.  Can you resolve the spelling errors found by R CMD check? Add the word to `inst/WORDLIST` if it's a false positive.
```
Potential spelling errors:
  WORD   FOUND IN
grey   theme_clean.Rd:16
Zdeb   theme_clean.Rd:34
```

File: R/tableau.R (hunk 1 above)
Comment (vadimus202) [reply R/tableau.R]: It seems that all the parameters native to this function are already explicitly defined. The only remaining parameters being pulled from `scale_colour_hue` are the ones being passed to `discrete_scale` via the dots. 

I wasn't used to seeing that in other color palettes like `ggplot2::scale_color_brewer()` or `ggplot2::scale_color_viridis_d()`. But, I can see now that it's the intended format,  and taking it out makes it inconsistent with the rest of discrete color scales in `ggthemes`. 

I can put it back in, unless it's easier for you to discard that one deletion on your end. Let me know.

Thanks

**Comment by jrnold** [comment]
Thanks!  The new version with your addition was accepted by CRAN and should be up soon.

**Pull request merged**